client/src/components/SignUpContainer.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Button, Paper, TextField, Typography, Grid } from "@material-ui/core";
+
+const useStyles = {
+  root: {
+    padding: "10vh",
+    backgroundColor: "#6E3ADB",
+    height: "80vh"
+  },
+  text: {
+    fontWeight: "800",
+    fontSize: "4vh"
+  },
+  signUp: {
+    width: "50vw",
+    padding: "5vh",
+    textAlign: "center",
+    rounded: true,
+    backgroundColor: "white",
+    margin: "auto"
+  }
+};
+
+const SignUpContainer = props => {
+  const classes = useStyles;
```

Comment:
we should actually use the hooks API here, so notice the example using the makeStyles function around useStyles

Reply:
when I use make styles I'm getting this error "The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX." I'm not sure what I'm doing differently from the example

Reply:
when I log out the classes object it look like this {root: "makeStyles-root-1", signUp: "makeStyles-signUp-2"}, so the makeStyles hook is converting these object into strings

Reply:
That's right, so the nuance is below you are using `style` as the HTML attribute, but you should be using `className` if you are using the hooks